### PR TITLE
Add use case in developer guide

### LINKED ISSUE #14: Add use cases to DG
Give use cases (textual form) for a few representative user stories that need multiple steps to complete. e.g. Adding a tag to a person (assume the user needs to find the person first)

### PR DESCRIPTION
**Pull request by astraxq**
## Changes
- Add update student/task
- Add delete task

## Issues
- Resolves #14

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -141,6 +169,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
```
Comment (toh-xinyi) [inline docs/DeveloperGuide.md:169]: Line above this, should change AddressBook to MATHUTORING

**Review by toh-xinyi — COMMENTED** [review]
LGTM, just small change to be made. 👍

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -116,16 +116,44 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  Tutor requests to list students
+2.  MATHTUTORING shows a list of students
+3.  Tutor requests to delete a specific student in the list
+4.  MATHTUTORING deletes the student
```
Comment (gwynethguo) [inline docs/DeveloperGuide.md:128]: Should we add a period at the end of each line? I think it's optional (AB3 doesn't use periods but the textbook does) but perhaps if you want to, can make it uniform with the lines below??? :DDDDD

**Review by gwynethguo — COMMENTED** [review]
Overall, LGTM!! がんばってください！

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (astraxq) [reply docs/DeveloperGuide.md]: Done

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (astraxq) [reply docs/DeveloperGuide.md:169]: Done

**Pull request merged**